score-keeper.html

```diff
@@ -16,7 +16,7 @@ <h1><span id="p1Display">0</span>  to <span id="p2Display">0</span></h1>
 <button  id = "p2">Player Two</button>
 <button  id = "reset">Reset</button>
 
-<script type ="text/javascript" src = "scorekeeper.js"> </script>
+<script type ="text/javascript" src = "score-keeper.js"> </script>
```

Comment:
@ewal2 filenames shouldn't include whitespaces. If a file name has two parts we either use `-` or `_`. Also, the file extension should always be included `.html`, `.js`, `.css` etc.
In general:
`score keeper html` should be `score-keeper.html` etc.